package.json

```diff
@@ -130,13 +131,14 @@
     "js-beautify": "^1.7.5",
     "jsc-android": "236355.x.x",
     "jsdom-jscore": "git+https://github.com/iamcco/jsdom-jscore-rn.git#a562f3d57c27c13e5bfc8cf82d496e69a3ba2800",
+    "jsdom-jscore-rn": "git+https://github.com/iamcco/jsdom-jscore-rn.git#a562f3d57c27c13e5bfc8cf82d496e69a3ba2800",
```

Comment:
Interesting, do we have to have 2 lines/deps here even though they point to the very same commit?

Reply:
This was added automatically by `yarn upgrade`, probably because the package it's pointing at was renamed `jsdom-jscore-rn`.
I think we should keep `jsdom-jscore-rn` only and update the imports here https://github.com/wordpress-mobile/gutenberg-mobile/blob/develop/src/globals.js#L7
Updating 👍